### ISSUE TITLE
Merge into Salt?

### ISSUE THREAD

**Issue by techhat** [ministryofjustice/salt-cloud-provider-vcloud#2]
This looks really awesome, guys. Would you be interested in having it merged into Salt proper, to get more eyeballs on it? In any case, SaltStack would love to help with this driver in any way.

**Comment by ashb** [comment]
Yeah we'll be happy to merge it in once we test it a bit more heavily.

We also need to get some of the hacks we added on top of libcloud merged back upstream there first.

**Comment by ashb** [comment]
Example of one of the hacky bits I want to filter back upstream to libcloud:

https://github.com/ministryofjustice/salt-cloud-provider-vcloud/blob/d3708b3783d0027/salt/cloud/clouds/vcloud_support/libcloud_vcloud.py#L85-L91
